storage.py

```diff
@@ -105,10 +115,11 @@ def __init__(self, bot):
         while True:
             # use async sleep to avoid the bot locking up
             await asyncio.sleep(REPORT_INTERVAL_SECONDS)
-            if len(STORAGE) == 0:
+
+            if len(STORED_DRONES) == 0:
                 await storage_channel.send('No drones in storage.')
             else:
-                for stored in STORAGE:
+                for stored in STORED_DRONES:
                     # calculate remaining hours
                     remaining_hours = hours_from_now(
                         datetime.fromisoformat(stored.time))
```

Comment:
Maybe reword this/add a comment for better clarity? "stored.time" reads like "How long a drone has been stored for." Something like "release_time" would be clearer.